### ISSUE TITLE
Update plugin properties to declare them as multi-values property

### ISSUE THREAD

**Issue by ianwilliams1** [I8C/sonar-flow-plugin#28]
SonarQube LTS 8.9.0
sonar-flow-plugin-1.0

Plugin is throwing the following WARNING:

```

> WARN: Property 'sonar.flow.file.suffixes' is not declared as multi-values/property set but was read using 'getStringArray' method. The SonarQube plugin declaring this property should be updated.
> WARN: Property 'sonar.node.file.suffixes' is not declared as multi-values/property set but was read using 'getStringArray' method. The SonarQube plugin declaring this property should be updated.

```

As a result of SonarQube plugin API change, I believe file:
sonar-flow-plugin/src/main/java/be/i8c/codequality/sonar/plugins/sag/webmethods/flow/settings/[FlowLanguageProperties.java](https://github.com/I8C/sonar-flow-plugin/blob/master/sonar-flow-plugin/src/main/java/be/i8c/codequality/sonar/plugins/sag/webmethods/flow/settings/FlowLanguageProperties.java) needs to be updated to include:  `.multiValues(true)`

See similar [change for dotnet plugin](https://github.com/SonarSource/sonar-dotnet/pull/1111/files).
And similar [change for Cxx-plugin](https://github.com/SonarOpenCommunity/sonar-cxx/pull/1782/files)

This warning is an annoyance as it's an issue thrown by SonarQube in relation to the plugin, yet is displayed to the users who have no idea what it's about.
